src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {
```

Comment:
Better name is `FigureType`, since enum name should be singluar, but you already have entity type `Figure`